portfolio/src/main/webapp/style.css

```diff
@@ -1,18 +1,46 @@
 body {
-    background-color: seashell;
+    background-color: rgb(255, 245, 238);
 }
 
 .pImg {
-    max-width:10%;
+    max-width:5%;
```

Comment:
Why percent and not an explicit value in pixels? or something like em or rem which are relative to font size. Just curious, no need to change.